docs/install/integrations.md

```diff
@@ -127,20 +129,26 @@ dependencies {
     // ktlint will pick them up
 }
 
-task ktlint(type: JavaExec, group: "verification") {
+tasks.register("ktlintCheck", JavaExec) {
+    group = "verification"
     description = "Check Kotlin code style."
     classpath = configurations.ktlint
-    mainClass.set("com.pinterest.ktlint.Main")
-    args "src/**/*.kt"
+    mainClass = "com.pinterest.ktlint.Main"
+    args "src/**/*.kt", "!**/build/**"
```

Comment:
Also add "**.kts" like on line 68?

Reply:
We can.

Reply:
78de319793b7d9fab35a6bae409d65981d12ffa7